README.md

```diff
@@ -268,14 +268,13 @@ If you need to modify or add a direct runtime dependency declaration, follow the
 1) Make sure there are no pending changes to `requirements.txt` or `requirements.txt.in`
 2) Run `make requirements.txt`. This will update `requirements.txt` to the most recent releases of direct and transitive dependencies.
 3) Examine the changes to `requirements.txt` and commit them with a message like `Bumping dependencies`
-4) Make the desired change to `requirements.txt.in`
+4) If necessary, make the desired change to `requirements.txt.in`
 5) Run `make requirement.txt` again. This updates `requirements.txt` with direct and transitive entries related to your change.
```

Comment:
I thought your change eliminated the need for two commits.

Reply:
I don't think there was ever a requirement for two commits.  I've made changes in a single commit in the past.